config/sass-lint.yml

```diff
@@ -13,8 +13,9 @@
 # Linter SpaceBeforeBrace, option "allow_single_line_padding"
 # Linter VendorPrefix, option "identifier_list"
 
-files: 
+files:
   include: 'src/**/*.scss'
+  ignore: 'src/applications/proxy-rewrite/sass/style-consolidated.scss'
```

Comment:
dunno why but reinstalling our dependencies breaks sass lint on this file with a fatal error. i'm just excluding it from the linting for now and will write a follow-up bug for it.

Reply:
I was wondering for a long time why everything would break if a new engineer didn't have the right Yarn version.